src/Ennemies/Enemy.java

```diff
@@ -6,11 +8,11 @@
  * @author Maas Lalani
  * @version 1.2 2016-11-14
  */
-public class Enemy
+public abstract class Enemy
 {
     /* class fields */
     /** An array of possible enemy types. */
-    public static final String[] ENEMY_NAMES = { "Zombie", "Skeleton", "Warrior", "Goblin", "Werewolf", "Vampire" };
+    public static final String[] ENEMY_NAMES = { "El Grande", "The Little", "Hector The", "Explosive", "Strange", "Big and Fat" };
```

Comment:
Change to `public static final String[] ENEMY_NAMES = { "Zombie", "Skeleton", "Warrior", "Goblin", "Werewolf", "Vampire" };`

Reply:
ENEMY_NAMES was used here to add something like a nickname to the monster generated (El Grande Zombie, The Little Warrior, Hector the Vampire...)